### ISSUE TITLE
[DESIGN] LC 'How do you rate' Page Component 

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#14579]
## Issue Description
_Design needed for the 'How do you rate...' LC page component_

## Design Tasks
- Ensure the page is styled according to design spec. 
- Validate verbiage "Specific Terminology" 
- Design doesn't exist, design is to support FE build (entirely FE)  

## Acceptance Criteria
- [ ] Designer comments on this issue to approve page styling 
- [ ] Verbiage content is validated and approved

## Planning Considerations
- Design needed for FE (entirely FE)
- Section is called `How do you rate your experience on this page?`
- The user response options are: `Good` / `Bad` 
- After the user clicks on a response, the radio buttons should go away and a confirmation message display in its place. 
- The confirmation message should be: `Thank you for your feedback.` 
- Check in with Analytics team re backend integration. ([Ticket #13774](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13774) )

**Comment by andaleliz** [comment]
What are the possible errors with this feedback component? Thinking through a scenario where someone selects something, and it's not correctly submitted or there is a problem with the API...what should they do? Refresh the page and try again? Nothing?

It feels weird to say "thank you for your feedback" if we don't actually receive the feedback.

**Comment by jenniferlee-dsva** [comment]
Good point. I'm not sure. The confirmation message was for the scenario where their response was captured. (TBD Analytics team but it sounds like they're considering Medallia backend.)

If the response didn't go through, then I'm not sure what the better experience is. I'd be okay with nothing happens, so the user simply tries again. It seems like overkill to have a message, but I suppose if it's a persistent system failure, it could get annoying. 

Perhaps a simple message as you suggested in the event of a system fail: "Oops, something went wrong. Please refresh the page and try again."

**Comment by jenniferlee-dsva** [comment]
We can separate out the error scenarios if we need to from this ticket. The og ask was for the primary state - I had not thought of error scenarios.

**Comment by andaleliz** [comment]
That sounds good to me- we need to understand what the possible errors might be from Medallia, or whatever the team chooses, to provide the best UX for those scenarios.

**Comment by andaleliz** [comment]
![VA-LCT-Q+A Copy](https://user-images.githubusercontent.com/55992116/95508171-95e1d000-0980-11eb-88bc-a5b8770bd6dc.png)
![VA-LCT-Feedback Thank You](https://user-images.githubusercontent.com/55992116/95508180-98442a00-0980-11eb-861f-f69da97198bd.png)

How about leveraging our background-only alert component for this? We could then use red for the error state, especially if there is an action a user can/should take.

**Comment by jenniferlee-dsva** [comment]
For successful confirmation: I just don't think it needs color. The state change of `now-you-see-radio-buttons-now-you-don't-with-a-message `seems clear enough. Prefer simplicity. 

What did you have in mind for the error scenario? I guess color could be an aid - though I feel ambivalent/averse to red. After all, this isn't a required element/response/action on the user's part, so I'd rather not give it the feeling of critical must-do error message. 

Feeling wise - we're just letting them know, 'your click didn't go through, please try again - sorry, it's not you.' Know what I mean?

**Comment by jenniferlee-dsva** [comment]
I wish we could do something fun with the thank you though - like include an OK emoji or something. This is not REALLY our style though. Honestly, I think Veterans just like things simple and clear.

**Comment by andaleliz** [comment]
>I wish we could do something fun with the thank you though - like include an OK emoji or something. This is REALLY not our style though. Honestly, I think Veterans just like things simple and clear.

I'm with you. I played around with some fun ideas just for the exercise, but definitely not our style.

I chose the color route as I thought it would leave no doubt to the user that anything had changed.  A state change only is probably fine though, especially since this isn't a critical interaction. We can do that for any error messages we need as well. [Updated the mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1588167553516/2773c854e8ff3a725a5e3ee03272b78e8519f78b?wid=0-1602252163376) with a text only option.
